.travis.yml

```
@@ -1,47 +1,59 @@
-dist: trusty
+# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 
-sudo: false
-
-addons:
-  apt:
-    packages:
-      - unixodbc
-      - unixodbc-dev
-      - libmyodbc
-      - libsqliteodbc
-      - odbc-postgresql
+services:
+ - mysql
 
 os:
   - linux
+  - osx
+  - windows
+
+arch:
+  - x64
+  - x86
 
 julia:
-  - 1.0
+  - 1.3
+  - 1.4
   - nightly
 
 env:
-  - ODBC_TEST_DRIVERS="mysql, postgresql"
+  - JULIA_PROJECT="@."
 
 matrix:
-  fast_finish: true
+  exclude:
+    - os: osx
+      arch: x86
+    - os: linux
+      arch: x86
   allow_failures:
-    - julia: nightly
-
-services:
-  - mysql
-  - postgresql
+  - julia: nightly
+  include:
+    - stage: "Documentation"
+      julia: 1.5
+      os: linux
+      script:
+        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate(); Pkg.build("ODBC")'
+        - julia --project=docs/ docs/make.jl
+      after_success: skip
 
```

You can save some CI time by

1. not running the CI jobs on branches (so you have only one build here, instead of two)
2. caching the artifacts

```suggestion

branches:
  only:
  - master
  - gh-pages # For building documentation
  - /^testing-.*$/ # testing branches
  - /^v[0-9]+\.[0-9]+\.[0-9]+$/ # version tags

cache:
  directories:
  - $HOME/.julia/artifacts

```